react-components/table-intro/intro.md

```diff
@@ -1 +1,5 @@
 In this scenario, you will learn how to use PatternFly 4 React Table for the first time.
```

Comment:
I think remove "for the first time" here, because we don't know if its their first time.